ocean_lib/ocean/test/test_ocean_assets.py

```diff
@@ -163,6 +164,17 @@ def test_update_datatokens(
     assert _asset.datatokens[0].get("address") == old_datatokens[0].get("address")
     assert _asset.services[0].datatoken == old_datatokens[0].get("address")
 
+    nft_token = publisher_ocean_instance.get_nft_token(_asset.nft["address"])
+    bn = publisher_ocean_instance.web3.eth.block_number
+    tx_id = nft_token.get_event_log(
+        event_name="MetadataUpdated", from_block=bn, to_block=bn, filters=None
+    )[0].transactionHash
+    receipt = publisher_ocean_instance.web3.eth.get_transaction_receipt(tx_id)
+    validation_event = nft_token.events.MetadataValidated().processReceipt(
```

Comment:
Can we use here `get_event_log` instead of `processReceipt` and search for `MetadataValidated`?